.github/actions/spell-check/allow.txt

```diff
@@ -1 +1,5 @@
+bkmeneguello
```

Comment:
should these be added to the user file vs the allow list?

Reply:
I think this one is for names, since it's GitHub users. @jsoref , does this make more sense here on in expect?

Reply:
Eww. I messed up. That allow.txt file should be moved into the allow directory. And yes, that entry should be moved to names.txt

Reply:
No problem, we can work around it. :) 
Yes, I've checked and names.txt seems to not be working. But currently, this is the file where these additions make sense, right?

Reply:
If you move/delete the `allow.txt` file into the `allow/` directory, it'll work.

Reply:
I'm guessing out of the scope of this PR, but something we want to do. Thank you!